### PR TITLE
feat(icons): Update icons.yml 0916

### PR DESCRIPTION
**Pull request by LMapes**
Updated icons.yml with

aliases
friendly names

approved by BXD

#### Changelog

**Changed**

- icons.yml

changes made correspond to issue #9473

### CODE REVIEW

**Comment by jnm2377** [comment]
Looks like some tests are failing for this.

**Comment by LMapes** [comment]
@jnm2377 I don't know how to fix these errors.

**Comment by jnm2377** [comment]
Looks like it's breaking bc of bad indentation. I would double check your formatting to make sure there aren't other bad indentations besides this one. 
Screen Shot 2021-09-17 at 3 08 15 PM


Also, public tests might need to be updated. I'm not sure. But looking at the PR you linked, it definitely updated test after adding new icon data.
Screen Shot 2021-09-17 at 3 09 41 PM

File: packages/icons/icons.yml [hunk 1]
```diff
@@ -6762,8 +7204,14 @@
     - school
     - school of fish
 - name: fit-to-height
-  friendly_name: fit-to-height
-  aliases: []
+    friendly_name: Fit-to-height
+  aliases:
```
Comment (jnm2377) [inline packages/icons/icons.yml]:
```suggestion
  friendly_name: Fit-to-height
  aliases:
```

**Comment by LMapes** [comment]
@joshblack I don't know how to fix indentation. It looked visually okay to me in VS Code.

File: packages/icons/icons.yml [hunk 2]
```diff
@@ -8424,33 +9066,51 @@
   sizes:
     - 32
 - name: inspection
-  friendly_name: inspection
-  aliases: []
+    friendly_name: Inspection
+  aliases:
```
Comment (jnm2377) [inline packages/icons/icons.yml]:
```suggestion
  friendly_name: Inspection
  aliases:
```

File: packages/icons/icons.yml [hunk 3]
```diff
@@ -10121,13 +10885,24 @@
...
+    - file
+    - repository
+    - folder
   sizes:
     - 32
 - name: media-cast
-  friendly_name: media-cast
-  aliases: []
+    friendly_name: Media cast
+  aliases:
```
Comment (jnm2377) [inline packages/icons/icons.yml]:
```suggestion
  friendly_name: Media cast
  aliases:
```

File: packages/icons/icons.yml [hunk 4]
```diff
@@ -13781,13 +14732,22 @@
   sizes:
     - 32
 - name: settings--services
-  friendly_name: settings--services
-  aliases: []
+   friendly_name: Settings services
+  aliases:
```
Comment (jnm2377) [inline packages/icons/icons.yml]:
```suggestion
  friendly_name: Settings services
  aliases:
```

File: packages/icons/icons.yml [hunk 5]
```diff
@@ -13781,13 +14732,22 @@
...
+    - controls
+    - services
+    - gear
   sizes:
     - 32
 - name: settings--view
-  friendly_name: settings--view
-  aliases: []
+    friendly_name: Settings view
+  aliases:
```
Comment (jnm2377) [inline packages/icons/icons.yml]:
```suggestion
  friendly_name: Settings view
  aliases:
```

File: packages/icons/icons.yml [hunk 6]
```diff
@@ -16900,8 +18080,12 @@
   sizes:
     - 32
 - name: volume--block-storage
-  friendly_name: volume--block-storage
-  aliases: []
+    friendly_name: Volume block storage
+  aliases:
```
Comment (jnm2377) [inline packages/icons/icons.yml]:
```suggestion
  friendly_name: Volume block storage
  aliases:
```

File: packages/icons/icons.yml [hunk 7]
```diff
@@ -15061,8 +16047,12 @@
     - shop
     - marker
 - name: tank
-  friendly_name: tank
-  aliases: []
+    friendly_name: Tank
+  aliases:
```
Comment (jnm2377) [inline packages/icons/icons.yml]:
```suggestion
  friendly_name: Tank
  aliases:
```

Comment (jnm2377) [reply packages/icons/icons.yml]: cc: @LMapes these are the indentations that aren't correct

File: packages/icons/icons.yml [hunk 8]
```diff
@@ -15078,43 +16068,87 @@
...
+    - clipboard
+    - plus
+    - add
   sizes:
     - 32
 - name: task--approved
-  friendly_name: task--approved
-  aliases: []
+    friendly_name: Task approved
+  aliases:
```
Comment (jnm2377) [inline packages/icons/icons.yml]:
```suggestion
  friendly_name: Task approved
  aliases:
```

File: packages/icons/icons.yml [hunk 9]
```diff
@@ -15078,43 +16068,87 @@
...
+    - approved
+    - approve
+    - approval
   sizes:
     - 32
 - name: task--asset-view
-  friendly_name: task--asset-view
-  aliases: []
+ friendly_name: Task asset-view
+  aliases:
```
Comment (jnm2377) [inline packages/icons/icons.yml]:
```suggestion
  friendly_name: Task asset-view
  aliases:
```

File: packages/icons/icons.yml [hunk 10]
```diff
@@ -15078,43 +16068,87 @@
...
+    - clipboard
+    - pin
+    - location
   sizes:
     - 32
 - name: task--settings
-  friendly_name: task--settings
-  aliases: []
+   friendly_name: Task settings
+  aliases:
```
Comment (jnm2377) [inline packages/icons/icons.yml]:
```suggestion
  friendly_name: Task settings
  aliases:
```

File: packages/icons/icons.yml [hunk 11]
```diff
@@ -15078,43 +16068,87 @@
...
+    - task
+    - clipboard
+    - star
   sizes:
     - 32
 - name: task--tools
-  friendly_name: task--tools
-  aliases: []
+ friendly_name: Task tools
+  aliases:
```
Comment (jnm2377) [inline packages/icons/icons.yml]:
```suggestion
  friendly_name: Task tools
  aliases:
```

File: packages/icons/icons.yml [hunk 12]
```diff
@@ -15799,8 +16863,12 @@
   sizes:
     - 32
 - name: tool-kit
-  friendly_name: tool-kit
-  aliases: []
+ friendly_name: Tool kit
+  aliases:
```
Comment (jnm2377) [inline packages/icons/icons.yml]:
```suggestion
  friendly_name: Tool kit
  aliases:
```

File: packages/icons/icons.yml [hunk 13]
```diff
@@ -16270,8 +17362,12 @@
     - 24
     - 32
 - name: ungroup-objects
-  friendly_name: ungroup-objects
-  aliases: []
+    friendly_name: Ungroup objects
+  aliases:
```
Comment (jnm2377) [inline packages/icons/icons.yml]:
```suggestion
  friendly_name: Ungroup objects
  aliases:
```

**Comment by mjabbink** [comment]
@LMapes @jnm2377 Did the naming get resolved on this issue? I’m seeing lots of code names showing up in the library that I think this issue will resolve.

**Comment by LMapes** [comment]
@mjabbink I added all the metadata content for all the UI icons into the .yml. Why it's not displaying by now is unclear to me.

**Comment by jnm2377** [comment]
Hey @LMapes sorry if I didn't make it clear in the past comments. I brought up tests failing a while back, which need to be addressed before this PR can get merged. Typically, contributors are responsible for updating their PRs accordingly when we give feedback or leave reviews. I tried to help out by updating all of the indentation errors your code had, but looks like the snapshots still need to be updated since you've added new metadata.

File: packages/icons/icons.yml [hunk 14]
```diff
@@ -10047,13 +10790,34 @@
...
+    - directions
+    - diagram
+    - plan
+    - legend
   sizes:
     - 32
 - name: map--identify
-  friendly_name: map--identify
-  aliases: []
+  - map
```
Comment (tay1orjones) [inline packages/icons/icons.yml]: This line is the problem. Running the file through a [yaml validator](https://jsonformatter.org/yaml-formatter) returns the following error:

```
Error : bad indentation of a sequence entry at line 10812, column 3:
      - map
      ^
```

**Review by tay1orjones — CHANGES_REQUESTED** [review]

**Comment by LMapes** [comment]
@mjabbink I don't know how to correct this. Please assign a Carbon dev.

**Comment by Katie-A-IBM** [comment]
@joshblack @jnm2377 @tay1orjones 
Hi All, Would you be able to set up a call to discuss this issue?
Thanks
cc @LMapes

**Comment by tay1orjones** [comment]
@LMapes I'm happy to fix it, but don't know what the proper values should be. This is what it is currently:

```yml
- name: map--identify
  - map
    - travel
    - point
    - identify
    - ID
    - center
    - locate
    - location
    - graph
  sizes:
    - 32
```

There's no `friendlyName` or `aliases` entries. They were there previously but were removed as part of this PR. Should it be this instead?
```yaml
- name: map--identify
  friendly_name: map--identify
  aliases:
    - map
    - travel
    - point
    - identify
    - ID
    - center
    - locate
    - location
    - graph
  sizes:
    - 32
```

**Comment by LMapes** [comment]
@tay1orjones Thanks for clarifying what you need. This should work

- name: map--identify
  friendly_name: Map identify
  aliases:
    - map
    - travel
    - point
    - identify
    - target
    - exact
    - ID
    - center
    - locate
    - location
    - graph
  sizes:
    - 32

**Review by tay1orjones — APPROVED** [review]
I've corrected the `map--identify` metadata as well as another piece of indentation I found. There were also a few entries that had the `.svg` suffix added, which isn't needed and caused the build to fail.

**Review by jnm2377 — APPROVED** [review]

**Pull request merged**